modal.js

```diff
@@ -54,69 +52,63 @@ const data = [
   },
 ];
 
-seeProjectBtn.forEach((btn) => {
-  btn.addEventListener('click', () => {
-      console.log('click');
-    popUp.style.display = 'block';
-    modal.style.display = 'block';
-  });
-});closeModal.addEventListener('click', () => {
-  console.log('remove');
+closeModal.addEventListener('click', () => {
   popUp.style.display = 'none';
   modal.style.display = 'none';
 });
 
-const card = data.map((elem) => {
-  const modalHtml = `
-<div class="popup">
-        <div class="modal">
-          <div class="modal-div-1">
-            <h2 class="modal-title">${elem.name}</h2>
-            <ion-icon class="close-modal" name="close-outline"></ion-icon>
-          </div>
-          <div class="modal-div-2">
-            <p class="canopy">${elem.canopy}</p>
-            <p class="back-end">
-              <img class="counter" src="images/Counter.png" alt="counter" />
-              Back End Dev
-            </p>
-            <p class="back-end">
-              <img class="counter" src="images/Counter.png" alt="counter" />
-              2015
-            </p>
-          </div>
-          <div class="modal-div-3">
-            <div class="modal-img">
-              <img class="snapshoot1-mob" src="${elem.image1}" alt="snapshoot1" />
-              <img class="snapshoot1-desk" src="${elem.image2}" alt="snapshoot1" />
+for (let i = 0; i < data.length; i += 1) {
+  projectsList.innerHTML = `${projectsList.innerHTML}
+   <div class="content">
+            <div class="snapshoot">
+              <img
```

Comment:
- [ ] [OPTIONAL] it would be better if you used the newer ES6 syntax to loop through the data object. it makes it more readable and concise.